invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCommunitiesList.js

```diff
@@ -45,7 +44,7 @@ export class RecordCommunitiesList extends Component {
         ?.slice(0, maxDisplayedCommunities)
         .map((community) => (
           <Item key={community.id}>
-            <Image size="mini" src={community.links.logo} />
+            <Image size="mini" src={community.links.logo} alt="" />
```

Comment:
```suggestion
            <Image size="mini" src={community.links.logo} alt={i18next.t("Community logo")} />
```

Reply:
I don't think this is needed, as the logo doesn't have any functionality and doesn't provide any information, ref https://www.w3.org/WAI/tutorials/images/